uint8 slave_id
uint8 index
uint8 subindex
uint8[] value
string type

